solr/core/src/test-files/solr/configsets/cloud-minimal-field-limiting/conf/solrconfig.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" ?>
+
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!-- Minimal solrconfig.xml with /select, /admin and /update only -->
+
+<config>
+
+  <dataDir>${solr.data.dir:}</dataDir>
+
+  <directoryFactory name="DirectoryFactory"
+                    class="${solr.directoryFactory:solr.NRTCachingDirectoryFactory}"/>
+  <schemaFactory class="ClassicIndexSchemaFactory"/>
+
+  <luceneMatchVersion>${tests.luceneMatchVersion:LATEST}</luceneMatchVersion>
+
+  <updateHandler class="solr.DirectUpdateHandler2">
+    <commitWithin>
+      <softCommit>${solr.commitwithin.softcommit:true}</softCommit>
+    </commitWithin>
+    <updateLog class="${solr.ulog:solr.UpdateLog}"></updateLog>
+  </updateHandler>
+
+  <requestHandler name="/select" class="solr.SearchHandler">
+    <lst name="defaults">
+      <str name="echoParams">explicit</str>
+      <str name="indent">true</str>
+      <str name="df">text</str>
+    </lst>
+  </requestHandler>
+
+  <updateProcessor class="solr.NumFieldLimitingUpdateRequestProcessorFactory" name="max-fields">
+    <int name="maxFields">${solr.test.maxFields:1234}</int>
+  </updateProcessor>
```

Comment:
why declare this separately instead of in the one chain that uses it?  It's not obvious, when looking at the chain, that there is this additional processor.

Reply:
Having no-config vs config doesn't conceptually change where the <updateProcessor> element can be placed.
ex:
```<updateProcessor blah lbah />```
vs
```
<updateProcessor blah blah>
  <int blah blah/>
</updateProcessor>
```
Can be in the same places.
I went to the ref guide's [page on URPs](https://solr.apache.org/guide/solr/latest/configuration-guide/update-request-processors.html#custom-update-request-processor-chain) and observe the first example shows a chain with several URPs, one of which has config.  I think it's natural to place URPs inline in the chain.  Separating it out is for composability if there are multiple chains and URPs and you want to avoid possible repetition.

Reply:
I've updated this to be in line with your request.

Though I'm still curious if there's a reason to "prefer" this going forward, or whether it's subjective.  I've not paid much attention historically to how these appear in our configsets, but naively I'd assume that more composability is better.